benchmarking/utils/utilities.py

```diff
@@ -21,6 +22,33 @@
 from .custom_logger import getLogger
 
 
+def getBenchmarks(bfile, framework=None):
```

Comment:
this doesn't look like a generic function that belongs to utilities. Where is it copied from?

Reply:
This is a method from internal under benchmarking/utils/utils.py